numba/core/typing/builtins.py

```diff
@@ -40,30 +40,30 @@ def generic(self, args, kws):
 
 @infer_global(abs)
 class Abs(ConcreteTemplate):
-    int_cases = [signature(ty, ty) for ty in sorted(types.signed_domain)]
-    uint_cases = [signature(ty, ty) for ty in sorted(types.unsigned_domain)]
-    real_cases = [signature(ty, ty) for ty in sorted(types.real_domain)]
+    int_cases = [signature(ty, ty) for ty in sorted(types.np_signed_domain)]
```

Comment:
Yes. There are module level initializations and need to have an alias to retain their functionality as we introduce the new type system. As of now Numpy is the only 'complete' type system being able to mirror our old type system.

Reply:
OK, thanks for the explanation. I think this may require having to "live with this" for now to make progress but anticipate that it will need revisiting once things become more concrete.